pkg/workloadmeta/types.go

```diff
@@ -432,6 +436,7 @@ type KubernetesPod struct {
 	QOSClass                   string
 	KubeServices               []string
 	NamespaceLabels            map[string]string
+	HostNetwork                bool
```

Comment:
These fields could probably all use a brief bit of doc -- this one at least is not clear to me from its name.